### PR TITLE
[feat8]: UI Changes as per new production

### PR DESCRIPTION
**Pull request by Abhishek-Jain-1925**
[feat8]: UI Changes as per new production

Workdone are as follows- 
1. All Description fields have minLen=50 and maxLen=Infinity.
2. Project->Role field should be open input text field.
3. Profile->Designation should be open input text field instead of Dropdown.

### CODE REVIEW

File: src/components/Builder/Achievement/index.jsx [hunk 1]
```diff
@@ -295,7 +295,7 @@ const Achievement = ({ achievementData }) => {
                     <Input.TextArea
                       placeholder="Please provide a basic overview of the above achievement"
                       showCount
-                      maxLength={300}
+                      minLength={50}
```
Comment (Payal13) [inline src/components/Builder/Achievement/index.jsx:315]: what is max length we support in backend ? is it varchar field in db ?

Comment (Payal13) [reply src/components/Builder/Achievement/index.jsx:315]: its text field, so no problem with length

File: src/components/Resume/index.js [hunk 2]
```diff
@@ -191,7 +191,7 @@ const Resume = forwardRef(({ data }, ref) => {
       givenYear === currDate.getFullYear() &&
       givenMonth === currDate.getMonth()
     ) {
-      return " Present";
+      return "present";
```
Comment (Payal13) [inline src/components/Resume/index.js]: move this in constant and use everywhere.
Present. it should start with Cap's only

File: src/components/Builder/Experience/index.jsx [hunk 3]
```diff
@@ -256,6 +256,12 @@ const Experience = ({ experienceData }) => {
     }
   };
 
+  const tabStyle = {
```
Comment (Payal13) [inline src/components/Builder/Experience/index.jsx]: remove

Comment (Abhishek-Jain-1925) [reply src/components/Builder/Experience/index.jsx]: Done!

File: src/components/Resume/index.js (hunk 2 above)
Comment (Abhishek-Jain-1925) [reply src/components/Resume/index.js]: Done!

**Review by Payal13 — APPROVED** [review]

**Pull request merged**